sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/__init__.py

```diff
@@ -27,8 +27,8 @@
     SentenceSentiment,
     SentimentConfidenceScores,
     MinedOpinion,
-    AspectSentiment,
-    OpinionSentiment,
+    TargetSentiment,
```

Comment:
can you add a changelog with the OM breaking changes

Reply:
I've updated the changelog.  Let me know if there's anything else I should add there.